fastlane/Fastfile

```diff
@@ -0,0 +1,25 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Prepare for release"
+  lane :release  do |options|
+    version_bump_podspec(path: "Bow.podspec", version_number: options[:version_number])
```

Comment:
@miguelangel-dev I know you mentioned updating podsec file is a manual step but I can do it via this lane using the version number. Do you think that would be a problem?

Reply:
That should not be a problem. However, I think we would need to bump the other podspecs in the repo as well. There are many other modules and we keep them in sync with the same version number.